engine/src/main/java/org/terasology/rendering/dag/nodes/OutlineNode.java

```diff
@@ -76,9 +125,47 @@ public OutlineNode(Context context) {
     public void process() {
         PerformanceMonitor.startActivity("rendering/outline");
 
+        // Common Shader Parameters
+
+        outlineMaterial.setFloat("viewingDistance", renderingConfig.getViewDistance().getChunkDistance().x * 8.0f, true);
+
+        outlineMaterial.setFloat("daylight", backdropProvider.getDaylight(), true);
+        outlineMaterial.setFloat("tick", worldRenderer.getMillisecondsSinceRenderingStart(), true);
+        outlineMaterial.setFloat("sunlightValueAtPlayerPos", worldRenderer.getTimeSmoothedMainLightIntensity(), true);
+
+        cameraDir = activeCamera.getViewingDirection();
+        cameraPosition = activeCamera.getPosition();
+
+        outlineMaterial.setFloat("swimming", activeCamera.isUnderWater() ? 1.0f : 0.0f, true);
+        outlineMaterial.setFloat3("cameraPosition", cameraPosition.x, cameraPosition.y, cameraPosition.z, true);
+        outlineMaterial.setFloat3("cameraDirection", cameraDir.x, cameraDir.y, cameraDir.z, true);
+        outlineMaterial.setFloat3("cameraParameters", activeCamera.getzNear(), activeCamera.getzFar(), 0.0f, true);
+
+        sunDirection = backdropProvider.getSunDirection(false);
+        outlineMaterial.setFloat3("sunVec", sunDirection.x, sunDirection.y, sunDirection.z, true);
+
+        outlineMaterial.setFloat("time", worldProvider.getTime().getDays(), true);
+
+        // Specific Shader Parameters
+
+        outlineMaterial.setFloat("texelWidth", 1.0f / sceneOpaqueFboWidth);
+        outlineMaterial.setFloat("texelHeight", 1.0f / sceneOpaqueFboHeight);
+
+        outlineMaterial.setFloat("pixelOffsetX", pixelOffsetX);
+        outlineMaterial.setFloat("pixelOffsetY", pixelOffsetY);
+
+        // Actual Node Processing
+
         renderFullscreenQuad();
 
         PerformanceMonitor.endActivity();
 
     }
+
+    @Override
+    public void update() {
```

Comment:
Oh, nevermind my previous comment. I was surprised at my slopiness, and went ahead and looked at the commits.. and I did nothing foolish :P
Earlier, the Node did not have any internal FBOs stored, since that task was allocated to the ShaderParameters class, which retrieved it every frame. However, now that SP class does not exist, this Node has to handle the FBO, and hence it is subscribing.
So no, we didn't neglect it :)